Dockerfile.local

```diff
@@ -82,14 +83,16 @@ RUN set -x \
    && mv sonar-scanner-*-linux sonarscanner-cli-linux \
    && mv sonar-scanner-*-macosx sonarscanner-cli-macosx \
    # Install skope
-   && wget -q -O skopeo-linux.tar.gz 'https://bintray.com/screwdrivercd/screwdrivercd/download_file?file_path=skopeo-1.0.0-linux.tar.gz' \
+   && wget -q -O skopeo-linux.tar.gz 'https://github.com/screwdriver-cd/sd-packages/releases/download/v0.0.30/skopeo-linux.tar.gz' \
    && tar -C . -ozxvf skopeo-linux.tar.gz \
    && chmod +x skopeo \
    # Install zstd
-   && wget -q -O zstd-cli-linux.tar.gz 'https://bintray.com/screwdrivercd/screwdrivercd/download_file?file_path=zstd-cli-1.4.8-linux.tar.gz' \
-   && wget -q -O zstd-cli-macosx.tar.gz 'https://bintray.com/screwdrivercd/screwdrivercd/download_file?file_path=zstd-cli-1.4.8-macosx.tar.gz' \
+   && wget -q -O zstd-cli-linux.tar.gz 'https://github.com/screwdriver-cd/sd-packages/releases/download/v0.0.30/zstd-cli-linux.tar.gz' \
```

Comment:
Does it look better this way? Or is it better to not automatically track the latest?
`https://github.com/screwdriver-cd/sd-packages/releases/latest/download/zstd-cli-linux.tar.gz`

Reply:
Since this file is for local build, I think it can be latest, but I would like to follow the command of the real Dockerfile without any deep meaning.
https://github.com/screwdriver-cd/launcher/blob/006a6b17eb4d685469e06265772fec0e83e0f105/Dockerfile#L112

If you like latest at build time, you can edit the local file.